packages/crayons-core/src/components/checkbox/checkbox.scss

```diff
@@ -20,14 +21,16 @@
   }
 }
 
-.label-field {
+#description {
   font-size: 12px;
   color: #475867;
   letter-spacing: 0;
-  line-height: 20px;
+  line-height: 18px;
   position: relative;
-  padding-left: 28px;
+  padding-left: 22px;
   font-weight: 400;
+  margin-bottom: 10px;
```

Comment:
What's this margin for? Also why `10px`? Chandan mentioned that such white-space between elements should be in increments of `8px`. Let's ensure this. This spacing is best applied to the entire wrapper (i.e. `fw-checkbox`/`:host`) itself.

Reply:
Yes Manoj, the 10px margin was added before we got confirmation on the standards. Have changed it to 8px now as per the standards.